File: Atrasti.API/Models/Profile/CompanyPage_Res.cs
using System.Collections.Generic;
using Atrasti.API.Models.Company;
using Atrasti.API.Models.ProductModels;

namespace Atrasti.API.Models.Profile
{
    public class CompanyPage_Res
    {
        public ICollection<Product_Res> Products { get; set; } 
        public Company_Res Company { get; set; }
        public CompanyInfo_Res CompanyInfo { get; set; }
    }
}

File: Atrasti.ReactNative/Atrasti/src/Api/Models/Responds/CompanyPage_Res.ts

import { Product_Res } from './Product_Res';
import { Company_Res } from './Company_Res';
import { CompanyInfo_Res } from './CompanyInfo_Res';

export interface CompanyPage_Res {
    products: Product_Res[];
    company: Company_Res;
    companyInfo: CompanyInfo_Res;
}